### ISSUE TITLE
Autoencoder predicts every image pure white.

### ISSUE THREAD

**Issue by tavishm** [keras-team/keras#14564]
This really isn't a bug, but I thought asking it here would be appropriate(tell me if I'm wrong). I'm making an auto encoder, trying image recreation. The input image dimensions are (256,256,3). Here's the model architechture: 
```
Model: "model"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
input_1 (InputLayer)         [(None, 256, 256, 3)]     0         
_________________________________________________________________
conv2d (Conv2D)              (None, 256, 256, 12)      156       
_________________________________________________________________
batch_normalization (BatchNo (None, 256, 256, 12)      48        
_________________________________________________________________
conv2d_1 (Conv2D)            (None, 256, 256, 12)      588       
_________________________________________________________________
conv2d_2 (Conv2D)            (None, 256, 256, 3)       327       
=================================================================
Total params: 1,119
Trainable params: 1,095
Non-trainable params: 24
_________________________________________________________________

```

It's small, but I've tried upto 58 million trainable params. Doesn't make a difference. The images are of leaves, with a contrasting background(normally something close to white). The background is normally 30-50 % of the image. loss function is mse, tried binary_crossentropy(most blogs used that, I didn't get why) and kldivergence. Using adam, but tried adagrad and sgd. I get 85-90% accuracy, but its always a white image, or rarely some recreation, but it's far from the original.

Please help.

**Comment by bguetarni** [comment]
Please share your code or at least a notebook so that we can get an idea of what is going on in your code.
Thanks.

**Comment by jvishnuvardhan** [comment]
@tavishm Sorry for the late response. Is this still an issue for you?

Can you please share a simple standalone code to reproduce the issue? Thanks!

If this is not a bug, then [`TF Forum`](https://discuss.tensorflow.org/) is the right place to post this issue as there is a big community support and learn from each other. Thanks!

**Comment by tavishm** [comment]
Okay, I'll look into it.Sent from my Galaxy
-------- Original message --------From: Vishnuvardhan Janapati ***@***.***> Date: 9/29/21  1:47 AM  (GMT+05:30) To: keras-team/keras ***@***.***> Cc: Tavish Mankash ***@***.***>, Mention ***@***.***> Subject: Re: [keras-team/keras] Autoencoder predicts every image pure white. (#14564) 
@tavishm Sorry for the late response. Is this still an issue for you?
Can you please share a simple standalone code to reproduce the issue? Thanks!
If this is not a bug, then TF Forum is the right place to post this issue as there is a big community support and learn from each other. Thanks!
